<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttPar</class>
 <widget class="QMainWindow" name="AttPar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AttPar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>951</width>
      <height>312</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="readInputB">
         <property name="text">
          <string>Read From Input File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="inputT">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0.8933	-0.4470	-0.0477;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;0.1744	0.2467	0.9533;&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;-0.4144	-0.8598	0.2983&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Convert From:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Principal Angle/Axis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DCM - Direction Cosine Matrix</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Quaternions</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Eular Angles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gibbs Vector</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cayley-Klein</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>To:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <item>
          <property name="text">
           <string>DCM - Direction Cosine Matrix</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Principal Angle/Axis</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Quaternions</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Eular Angles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Gibbs Vector</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cayley-Klein</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Command History:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="history"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="convB">
           <property name="text">
            <string>Perform Conversion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Send to File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="outputT">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Base:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="base"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="height"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Area:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="area"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="calc_area_b">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>551</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Area of Triangle Using MATLAB</string>
    </property>
   </widget>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>calc_area_b</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
